data/events/2024-belo-horizonte.yml

```
@@ -0,0 +1,131 @@
+name: "2024-belo-horizonte" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "Belo Horizonte" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysbh" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+#description: "Em breve!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2024-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2024-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-09-28T00:00:00-03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-09-28T23:59:59-03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+  
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start:  # start accepting talk proposals.
+cfp_date_end:  # close your call for proposals.
+cfp_date_announce:  # inform proposers of status
+
+cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+registration_date_end: # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# masthead_background: "bh.png"
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+#location: "Nova Lima" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
+  - name: organizadores
+  - name: código de conduta
+    url: /events/2024-belo-horizonte/conduta 
+  - name: requisitos para as palestras
+    url: /events/2024-belo-horizonte/propose 
```

If you're ready to accept proposals, you'll need to set the `cfp_date` variables on lines 17–19 (you'll also want to include submission details in that page). If you're _not_ ready to accept proposals, you should comment this out.

---

We're making code changes to the site and in an effort to speed things along, I'm going to go ahead and comment this out for now.
```suggestion
  # - name: requisitos para as palestras
  #   url: /events/2024-belo-horizonte/propose 
```